musicname/zs_starrylit_haven_v11.cfg

```
@@ -0,0 +1,9 @@
+"music"
+{
+    "hipshop.mp3"                           "Deltarune OST - Hip Shop"
+  //"meteorhouse.mp3"                       ""
+    "troianbeauty.mp3"                      "FF IV OST - Troian Beauty"
```

I assume this is Final Fantasy 4.
```suggestion
    "troianbeauty.mp3"                      "FFIV OST - Troian Beauty"
```